tests/trainer/test_trainer.py

```diff
@@ -3451,3 +3456,103 @@ def test_hyperparameter_search_backends(self):
             list(ALL_HYPERPARAMETER_SEARCH_BACKENDS.keys()),
             list(HPSearchBackend),
         )
+
+
+@require_peft
+@require_torch
+class TrainerPeftTest(unittest.TestCase, TrainerIntegrationCommon):
+    def get_regression_adapter_trainer(
+        self,
+        keep_report_to=False,
+        adapter_name="default",
+        adapter_args=None,
+        **kwargs,
+    ):
+        # from transformers import OPTForCausalLM
+        base_model_id = "hf-internal-testing/tiny-random-OPTForCausalLM"
+        pretrained_base_model = AutoModelForCausalLM.from_pretrained(base_model_id).to(torch_device)
+        tokenizer = AutoTokenizer.from_pretrained(base_model_id)
+
+        def m(sample):
+            return tokenizer(
+                sample["quote"],
+                return_tensors="pt",
+                padding="max_length",
+                truncation=True,
+                max_length=tokenizer.model_max_length,
+            )
+
+        from datasets import load_dataset
+
+        data = load_dataset("Abirate/english_quotes")
+        data = data["train"].filter(lambda example, indice: indice < 9, with_indices=True).map(m, batched=True)
+
+        # generate peft
+        from peft import LoraConfig, get_peft_model
+
+        adapter_args = adapter_args or {
```

Comment:
Should test for none-ness - otherwise an empty dict will be overridden 
```suggestion
        adapter_args = adapter_args if adapter_args is not None else {
```